cpp/tests/io/csv/csv_test.cu

```diff
@@ -42,6 +42,18 @@ bool checkFile(const char *fname)
     return (stat(fname, &st) ? 0 : 1);
 }
 
+char* getTempFile(const char *fname)
```

Comment:
Suggest naming this `get_temp_filename`. Note filename not file since that's what it returns, and underscore spelling since we don't use camel case in libcudf...